pkgs/applications/networking/browsers/ladybird/default.nix

```diff
@@ -1,71 +1,117 @@
 { lib
 , stdenv
 , fetchFromGitHub
+, fetchurl
+, fetchzip
 , cmake
 , ninja
-, unzip
 , wrapQtAppsHook
 , libxcrypt
 , qtbase
+, qtmultimedia
 , nixosTests
 }:
 
+let
+  cldr_version = "43.1.0";
+  ucd_version = "15.0.0";
+  emoji_txt_version = "15.0";
+  tzdb_version = "2023c";
+  cacert_version = "2023-05-30";
+  cldr = fetchzip {
+    url = "https://github.com/unicode-org/cldr-json/releases/download/${cldr_version}/cldr-${cldr_version}-json-modern.zip";
+    stripRoot = false;
+    hash = "sha256-+PCYYUWwiDyTfDvSxBAbVRpA7YkuQGKr3ZPlnc9PnKk=";
+  };
+  ucd = fetchzip {
+    url = "https://www.unicode.org/Public/${ucd_version}/ucd/UCD.zip";
+    stripRoot = false;
+    hash = "sha256-jj6bX46VcnH7vpc9GwM9gArG+hSPbOGL6E4SaVd0s60=";
+  };
+  emoji_txt = fetchurl {
+    url = "https://www.unicode.org/Public/emoji/${emoji_txt_version}/emoji-test.txt";
+    hash = "sha256-hEXyOsg4jglr4Z0CYuFPzv+Fb/Ugk/I1bciUhfGoU9s=";
+  };
+  tzdb = fetchzip {
+    url = "https://data.iana.org/time-zones/releases/tzdata${tzdb_version}.tar.gz";
+    stripRoot = false;
+    hash = "sha256-G/IC0EHQcuXUTr2G/6Kn1giC0d1WC9hHIE2fQI26rlM=";
+  };
+  cacert = fetchurl {
+    url = "https://curl.se/ca/cacert-${cacert_version}.pem";
+    hash = "sha256-X63K6QqkrgQRUPji0mw32YBSLNtJ+SP8HhteuNdOca0=";
+  };
```

Comment:
In general these should probably be dependencies on other packages wherever possible

Reply:
hm ok. 

looks like we could use `unicode-character-database` for the UCD download, the versions match (atm). But, the build step would still need to have the `version.txt` file in the location of the database at build time, and not at runtime.

there's a `tzdata` package as well, but I'm not sure whether it actually installs the raw extracted files, or simply compiles them into a more distro-usable format.

The `cldr-annotations` package doesn't use the JSON download of the CLDR release, and seems to only grab the "common" files, whereas we use more than just those.

we can definitely use the cacert package though

Reply:
I wonder if you could use the `.src` (or `.srcs`?) of tzdata?

Reply:
I would assume best effort with whatever compatible packages exist currently is fine, but a committer could have a different opinion. In that case I'll happily help get the other things available as individual packages

Reply:
I was able to eliminate all of the manual fetches except for the CLDR json by using existing packages.